go.mod

```diff
@@ -114,6 +114,6 @@ replace (
 	github.com/operator-framework/operator-lifecycle-manager => github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190605231540-b8a4faf68e36
 	k8s.io/helm => k8s.io/helm v2.14.1+incompatible
 	k8s.io/kube-state-metrics => k8s.io/kube-state-metrics v1.6.0
-	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.2.0-beta.3
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.2.0-rc.0
```

Comment:
Do we need a `replace` for this? Or can we update our `require` version to `v0.2.0-rc.0`?

Reply:
`require` directives will grab the [highest available, compatible version]( https://github.com/golang/go/wiki/Modules#version-selection) of a dependency >= the version specified. `replace` will force the version to be == the specified version. We should keep the `replace` in the SDK's top-level modfile IMO.

Reply:
I don't think that's right.

> The minimal version selection algorithm is used to select the versions of all modules used in a build. For each module in a build, the version selected by minimal version selection is always the semantically highest of the versions explicitly listed by a require directive in the main module or one of its dependencies.

So in this case, as long as no other depenency declares a higher semantic version dependency on `controller-runtime`, Go will _always_ choose `v0.2.0-rc.0` even when future versions are released.

Reply:
Clarification: Go only chooses the highest available version _when the dependency is new_; otherwise the version is locked. Using `require` is sufficient here.